angular/projects/admin-nrpti/src/app/mines/mines-collection-detail/mines-collection-detail-record-row/mines-collection-detail-record-row.component.html

```diff
@@ -1,4 +1,6 @@
-<td scope="row" data-label="Name" class="col-6">{{ rowData.recordName || '-' }}</td>
+<td scope="row" data-label="Name" class="col-6 d-flex">{{ rowData.recordName || '-' }}
+  <i *ngIf="rowData.isLink" class="material-icons ml-2">link</i>
```

Comment:
For some reason sonarcloud is obsessed with changing i tags to em tags...

Reply:
Updated